src/components/00-materials/CHANGELOG.md

```diff
@@ -1,8 +1,7 @@
-- 3.0.0
-  - no breaking changes here. Release is needed for the other components.
+## 3.0.0
```

Comment:
In every other changelog there are 3 `###`. I would keep this number the same.

Reply:
Yes, the other changelogs are old. We defined the "new style" here: https://github.com/axa-ch/patterns-library/pull/1563/files

Reply:
Then we need an issue to change all other changelogs

Reply:
Issue created.

Reply:
Thanks